spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -119,7 +119,7 @@
       end
     end
     context "load_whitelist" do
-      it "should not be called if there was no whitelist config setting" do
+      it "not called if there was no whitelist config setting" do
```

Comment:
honestly, I feel like i've already spent too much time on this.  They don't read well, but they are understandable.  (I still have to update documentation, etc. for gdor-indexer)